src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobCleanerTask.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.ContainerCleaner;
+
+import java.util.stream.Stream;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class BlobCleanerTask {
+
+    private static final String TASK_NAME = "blob-cleaner";
+
+    private static final Logger logger = getLogger(BlobCleanerTask.class);
+
+    private final ContainerCleaner containerCleaner;
+    private final ServiceConfiguration serviceConfiguration;
+
+    public BlobCleanerTask(
+        ContainerCleaner containerCleaner,
+        ServiceConfiguration serviceConfiguration
+    ) {
+        this.containerCleaner = containerCleaner;
+        this.serviceConfiguration = serviceConfiguration;
+    }
+
+    public void run() {
+        logger.info("Started {} job", TASK_NAME);
+
+        getAvailableContainers().forEach(containerCleaner::process);
```

Comment:
Will we get a nice exception message in the logs when `getAvailableContainers` fails?